src/main/java/net/dv8tion/jda/api/sharding/DefaultShardManagerBuilder.java

```diff
@@ -518,6 +518,22 @@ public DefaultShardManagerBuilder setRawEventsEnabled(boolean enable)
         return setFlag(ConfigFlag.RAW_EVENTS, enable);
     }
 
+    /**
+     * Whether JDA should store the raw {@link net.dv8tion.jda.api.utils.data.DataObject DataObject} for every discord event, accessible through {@link net.dv8tion.jda.api.events.GenericEvent#getRawData() getRawData()}.
+     * <br>Warning: be aware that enabling this could consume a lot of memory if your event objects have a long lifetime.
+     * <br>Default: {@code false}
+     *
+     * @param  enable
+     *         True, if JDA should add the raw {@link net.dv8tion.jda.api.utils.data.DataObject DataObject} to every discord event.
+     *
+     * @return The DefaultShardManagerBuilder instance. Useful for chaining.
+     */
+    @Nonnull
+    public DefaultShardManagerBuilder setEventPassthrough(boolean enable)
+    {
+        return setFlag(ConfigFlag.EVENT_PASSTHROUGH, enable);
+    }
```

Comment:
Add an `@see` pointed at the new method on Event.
Also, we should probably add information to point to what _type_ of data they might see. Specifically, we should point them at the gateway docs

https://discord.com/developers/docs/topics/gateway#commands-and-events-gateway-events